app/assets/stylesheets/application.css.scss

```diff
@@ -9,9 +9,14 @@ body {
 }
 
 i {
+  &.huge   { font-size: 500%; }
   &.large  { font-size: 200%; }
   &.medium { font-size: 150%; }
   &.small  { font-size: $base-font-size; }
+
+  &.style1 {
+    color: #B8D30B;
```

Comment:
Color literals like `#B8D30B` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#B8D30B` should be written as `#b8d30b`